src/index.js

```diff
@@ -94,7 +94,7 @@ module.exports = (
       nodeEnv: false,
       minimize: false
     },
-    devtool: sourceMap ? "cheap-source-map" : false,
+    devtool: sourceMap ? "source-map" : false,
```

Comment:
What's the difference between `cheap-source-map` and `source-map`?

Reply:
I have absolutely no idea. I just thought it best not to go for the cheap option :P

Reply:
Looking at https://webpack.js.org/configuration/devtool/ I believe this is actually a fidelity thing :)

We could likely go back to cheap-source-map for performance.

Reply:
I've pushed a commit setting this back to cheap-source-map. Feel free to revert though if you feel otherwise, but perhaps perf should be a priority?

Reply:
@guybedford Can you measure the perf difference? If source-map is 80% slower than cheap-source-map then we might want to stick with cheap. But if source-map is 10% slower than cheap-source-map then we might want to use `source-map` for the fidelity.

Reply:
Just did a very quick test, and there were no significant differences between the two at all.